server/smc/smccli/smccli.go

```diff
@@ -2,14 +2,13 @@ package main
 
 import (
 	"fmt"
-	"io"
-	"os"
-
 	"go.dedis.ch/dela/cli/node"
-
 	db "go.dedis.ch/dela/core/store/kv/controller"
 	dkg "go.dedis.ch/dela/dkg/pedersen/controller"
 	minogrpc "go.dedis.ch/dela/mino/minogrpc/controller"
+	smcctl "go.dedis.ch/hbt/smc/smccli/controller"
+	"io"
+	"os"
```

Comment:
why change the import order ?

Normally we use the goimports convention, standard packages first.

Reply:
don't know: I see that Goland auto-magically re-order things, which is a PITT

Reply:
done